src/integration/pages/mediaAssetPage/arabicTC2/__snapshots__/canonical.test.js.snap

```diff
@@ -143,6 +143,26 @@ Object {
 }
 `;
 
+exports[`Canonical Media Asset Page I can see related content: "المايوه الشرعي" يثير جدلا في الساحل الشمالي في مصر 1`] = `"/arabic/tv-and-radio-53492142"`;
```

Comment:
Similarly, I don't know why these snapshots are here, these are related content / most watched. Think this ought to be fixed separately, and not part of this PR

Reply:
Integration snapshots are failing on latest; everyone else seems to be discarding the updates

Reply:
There's definitely something weird going on - I thought Andrew fixed them last week, so don't know why they are coming back.